### ISSUE TITLE
About formula and figure in PiraNet paper

### ISSUE THREAD

**Issue by HydrogenSulfate** [PredictiveIntelligenceLab/jaxpi#13]
In PiraNet paper, the input coordinates first embedded by `Coordinate Embedding` module, which represented by $\Phi(\mathbf{x})$ in paper, but in `PiraNet Block` $x$ is still used as input rather than $\Phi(\mathbf{x})$.

And I noticed that $x^{(l)}$ represent the input of l-th PiraNet block, so does $x^{(0)}$ represent the embedded coordinates $\Phi(\mathbf{x})$?

![image](https://github.com/PredictiveIntelligenceLab/jaxpi/assets/23737287/e13b575d-6f1a-44a4-9dff-41c7b2011536)

![image](https://github.com/PredictiveIntelligenceLab/jaxpi/assets/23737287/a28f6d03-28c3-4a7d-b566-c9ef3f9b18e4)

![image](https://github.com/PredictiveIntelligenceLab/jaxpi/assets/23737287/9088ea2c-baf2-4060-8a45-0c8ca02a03c8)

I have try piranet in paddle framework and it seems works well and get good accuracy in allen_cahn(l2_err= $8e-6$ , slightly better than reported in paper $2.24e-5$). That's a great work.

**Comment by sifanexisted** [comment]
Thank you for bringing this issue to our attention. We apologize for the error and will correct it in both the arXiv and published versions. 

It's great to hear that you're achieving even better results with Paddle!  Although I didn't fine-tune the hyperparameters too much, I understand how challenging it is to get below 1e-5. It is quite impressive!

**Comment by HydrogenSulfate** [comment]
> Thank you for bringing this issue to our attention. We apologize for the error and will correct it in both the arXiv and published versions.
> 
> It's great to hear that you're achieving even better results with Paddle! Although I didn't fine-tune the hyperparameters too much, I understand how challenging it is to get below 1e-5. It is quite impressive!

Thanks for reply, I just use sota config of allen cahn with two changes below:
1. replacing MLP with 3-block PirateNet(same as it in paper)
2. use grad norm weighting instead of NTK weighting, as the NTK matrix needs `[batch, len(param)]` matrix, too large to compute as paddle do not support vmap yet.

PirateNet seems has much quicker convergence speed than vanilla MLPs, I will try more examples with it!
